src/Version_Tool.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+class Version_Tool {
```

Comment:
Related, similar code in i18n-command:

https://github.com/wp-cli/i18n-command/blob/53518a11f314119e320597c7a8274f11b1295bdc/src/MakePotCommand.php#L462-L537

https://github.com/wp-cli/i18n-command/blob/53518a11f314119e320597c7a8274f11b1295bdc/src/FileDataExtractor.php

Maybe we can reuse some of that?